torchdynamo/variables/tensor.py

```diff
@@ -113,6 +113,12 @@ def create(cls, tx, proxy, example_value=None, nnmodule=None, **options):
             def wrap_fake_exception(func):
                 return func()
 
+        if (
+            proxy.node.target is torch.set_rng_state
+            or proxy.node.target is torch.get_rng_state
+        ):
+            unimplemented("rng state manipulation")
```

Comment:
Logically sound, but a little odd that this is on TensorVariable.create(). Let me patch this down and see what builder is doing...

Reply:
Yea - this is too far down. torch.* calls fallback to:

```
tensor_variable = TensorVariable.create(
                tx=tx,
                proxy=tx.output.create_proxy(
                    "call_function",
                    self.value,
                    *proxy_args_kwargs(args, kwargs),
                    current_tx=tx,
                ),
                **options,
            )
```

If we didn't figure out anything special for them.

Please (1) Move this unimplemented logic to `variables/torch.py` `def call_function`

and (2) Please make a dict of unimplemented_ops -> reason

then you can check something like

```
if self.value in unimplemented_ops:
     unimplemented(unimplemented_ops[self.value])
```